game-core/src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -111,7 +113,7 @@ public static File getUserMapsFolder() {
       mapsFolder.mkdirs();
     }
     if (!mapsFolder.exists()) {
-      ClientLogger.logError("Error, downloaded maps folder does not exist: " + mapsFolder.getAbsolutePath());
+      log.log(Level.SEVERE, "Error, downloaded maps folder does not exist: " + mapsFolder.getAbsolutePath());
```

Comment:
One of many examples where we could simplify to `log.severe`. To get this done and get it so we had just one kind of logging, the updates were pretty fast and furious. I'd like to leave such cleanups for future iterations. For now I'd like to get us at least using loggers consistently.